scripts/find_python_dependencies.py

```diff
@@ -0,0 +1,121 @@
+"""
+Spider and catalog dependencies.
+$ export OLIVE_DIRS=$(gittreeif origin/open-release/olive.master -q pwd)
+$ python find_deps.py $OLIVE_DIRS
+"""
+
+import concurrent.futures
```

Comment:
You can probably pare down these imports now that you've updated the code.

Reply:
I have updated it.